tests/common/test_gtsfm_data.py

```diff
@@ -30,13 +32,25 @@ class TestGtsfmData(unittest.TestCase):
     """Unit tests for GtsfmData."""
 
     def test_equality_with_same_data(self):
-        """Test equality with the same data."""
+        """Test equality with the same data (same object)."""
```

Comment:
its same value but not same object, right? We do a deepcopy.

Reply:
Good point, I'll update the comment

Reply:
fixed